diff -urN aa/vcdimager-0.7.20/frontends/xml/Makefile.am vcdimager-0.7.20/frontends/xml/Makefile.am
--- aa/vcdimager-0.7.20/frontends/xml/Makefile.am	2004-02-12 02:57:23.000000000 +0100
+++ vcdimager-0.7.20/frontends/xml/Makefile.am	2004-09-10 14:43:09.619948120 +0200
@@ -27,7 +27,7 @@
 
 MAINTAINERCLEANFILES = $(man_MANS)
 
-INCLUDES = -I$(top_srcdir) $(LIBPOPT_CFLAGS) $(LIBVCD_CFLAGS) $(XML_CFLAGS) $(LIBCDIO_CFLAGS) $(LIBISO9660_CFLAGS) $(XML_CFLAGS)
+INCLUDES = -I$(top_srcdir) $(LIBPOPT_CFLAGS) $(LIBVCD_CFLAGS) $(XML_CPPFLAGS) $(LIBCDIO_CFLAGS) $(LIBISO9660_CFLAGS) $(XML_CPPFLAGS)
 
 BUILT_SOURCES = videocd_dtd.inc
 MOSTLYCLEANFILES = videocd_dtd.inc
